Wire-iOS/Sources/Helpers/sound/AVSMediaManager+CustomSounds.swift

```diff
@@ -58,7 +58,7 @@ extension AVSMediaManager {
         playSound(MediaManagerSound.alert.rawValue)
     }
 
-    private func configureDefaultSounds() {
+    internal func configureDefaultSounds() {
```

Comment:
```suggestion
    func configureDefaultSounds() {
```

By default funcs are already internal